### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by cdaniluk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
+# terraform-aws-nexus [![](https://github.com/rhythmictech/terraform-aws-nexus/workflows/pre-commit-check/badge.svg)](https://github.com/rhythmictech/terraform-aws-nexus/actions) <a href="https://twitter.com/intent/follow?screen_name=RhythmicTech"><img src="https://img.shields.io/twitter/follow/RhythmicTech?style=social&logo=RhythmicTech" alt="follow on Twitter"></a>
```
Comment (sblack4) [inline README.md]: This is copypastaed from terraform-aws-nexus

File: README.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  vpc_id                         = "vpc-123456789012"
+}
+```
+
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+## Requirements
+
+| Name | Version |
+|------|---------|
+| terraform | >= 0.12.19 |
```
Comment (sblack4) [inline README.md:44]: This doesn't match the `.terraform-file` I'll put a suggestion to lower that file

File: .terraform-version [hunk 3]
```diff
@@ -0,0 +1 @@
+0.12.26
```
Comment (sblack4) [inline .terraform-version:1]:
```suggestion
0.12.19
```

match `version.tf`

File: backup.tf [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+  }
+}
+
+resource "aws_backup_selection" "this" {
+  count        = var.enable_efs_backups ? 1 : 0
+  name         = "${var.name}-EFS-backup"
+  iam_role_arn = aws_iam_role.backup[0].arn
+  plan_id      = aws_backup_plan.this[0].id
+  resources    = [aws_efs_file_system.this.arn]
+
```
Comment (sblack4) [inline backup.tf]:
```suggestion
```

extra space

File: bin/install-macos.sh [hunk 5]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+echo 'installing brew packages'
+brew install tfenv tflint terraform-docs pre-commit
+
+echo 'installing pre-commit hooks'
+pre-commit install
+
+echo 'installing terraform with tfenv'
+tfenv install
```
Comment (sblack4) [inline bin/install-macos.sh]: This is copypasta from before. Here's the new one https://github.com/rhythmictech/terraform-anycloud-template/blob/master/bin/install-macos.sh

**Review by sblack4 — APPROVED** [review]
:shipit:

File: .terraform-version (hunk 3 above)
Comment (smiller171) [reply .terraform-version:1]: The .terraform-version is only for tfenv, and we generally try to run the latest, not the minimum compatible. Minimum compatibility is there in part to set a baseline for ppl consuming the role.

File: bin/install-macos.sh (hunk 5 above)
Comment (sblack4) [reply bin/install-macos.sh]: This is also why the build is failing 
```
Terraform validate with tfsec............................................Failed
- hook id: terraform_tfsec
- exit code: 127

/Users/runner/.cache/pre-commit/repojehn6r5j/terraform_tfsec.sh: line 32: tfsec: command not found
/Users/runner/.cache/pre-commit/repojehn6r5j/terraform_tfsec.sh: line 32: tfsec: command not found
/Users/runner/.cache/pre-commit/repojehn6r5j/terraform_tfsec.sh: line 32: tfsec: command not found
/Users/runner/.cache/pre-commit/repojehn6r5j/terraform_tfsec.sh: line 32: tfsec: command not found
```

**Pull request merged**